src/lib/autoinstall/widgets/storage/partition_type.rb

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage"
+require "cwm/common_widgets"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # Determines the partition id
+      class PartitionType < CWM::ComboBox
+        # Constructor
+        def initialize
+          textdomain "autoinst"
+          super
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("Partition Type")
+        end
+
+        # @macro seeComboBox
+        def items
+          @items ||= [
+            ["", ""],
+            ["primary", "primary"],
+            ["logical", "logical"]
+          ]
+        end
+
+        # @macro seeAbstractWidget
+        def opt
```

Comment:
Why is this editable? The only acceptable values are primary and logical. Isn't it?

Reply:
It is. Fixed :)